### ISSUE TITLE
How to prepare the training data?

### ISSUE THREAD

**Issue by wangzhe0623** [milesial/Pytorch-UNet#34]
Hi, @changjo , thanks for your great job. I used this code training with my own data. I tried to train with more than labels. I have no idea how to prepare training data, so, can you post any example of data? such like colored images and ground truth images?  Thanks again~~~

**Comment by milesial** [comment]
The model accepts images with any number of channels, and should work with masks with more than one label, if you load and convert your masks correctly to numpy arrays (you would have to change the creation of the model, the `n_channels` and `n_classes`).

If you want examples of data, see what I trained it on : https://www.kaggle.com/c/carvana-image-masking-challenge/data

**Comment by wangzhe0623** [comment]
@milesial Thank again! I solved that!

**Comment by aimhabo** [comment]
@milesial whats meaning of _mask.gif? Is that a image of binary masks? or contains one object in each frame?

**Comment by milesial** [comment]
@aimhabo yes that is the suffix for all the masks, as images in black and white, so yes binary masks. They are not animated gifs with multiple frames, just single images.
